src/coach/defence/DefencePositionCoach.cpp

```diff
@@ -173,24 +185,32 @@ DefenderBot DefencePositionCoach::createBlockOnLine(const Field &field, const Po
 DefenderBot DefencePositionCoach::createBlockPass(const Field &field, PossiblePass &pass, const Vector2 &blockPoint) {
     DefenderBot bot;
     bot.type = botType::BLOCKPASS;
-    bot.blockFromID = pass.toBot.id;
+    bot.blockFromID = pass.toBot->getId();
     bot.targetPos = blockPoint;
     bot.orientation = pass.faceLine();
     return bot;
 }
-std::shared_ptr<Vector2> DefencePositionCoach::blockOnDefenseAreaLine(const Field &field, const PossiblePass &pass, const world::WorldData &world) {
-    auto visibleParts = FieldComputations::getVisiblePartsOfGoal(field, true, pass.endPos, world);
+
+std::shared_ptr<Vector2> DefencePositionCoach::blockOnDefenseAreaLine(const Field &field, const PossiblePass &pass) {
+    std::vector<Line> visibleParts = simulatedVisibleGoalParts(field, pass);
```

Comment:
It's quite hard to check if this behavior is still the same.

Reply:
I might be wrong but I think this is not the same behavior because previously only our bots were passed to the calculation.